docs/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+
+## [v12.1.0-rc.2] - 2025-08-28
+
+### 🚨 Breaking Changes
+- contracts/v2/sovereignChains/BridgeL2SovereignChain.sol: v1.0.0 -> v2.0.0
```

Comment:
@ignasirv it should increase only the minor versioning. No breaking changes added AFAIK